README.md

```diff
@@ -0,0 +1,69 @@
+Assisted Injection for JSR 330.
+============
+
+Manually injected dependencies for your JSR 330 configuration.
+
+Usage
+============
+```java
+class MyPresenter {
+  MyPresenter(Long foo, @Assisted String bar) {}
+  
+  @Assisted.Factory
+  interface Factory {
+    MyPresenter create(bar: String);
+  }
+}
+```
+This will generate the following:
+```java
+public final class MyPresenter_AssistedFactory implements MyPresenter.Factory {
+  private final Provider<Long> foo;
+  @Inject public MyPresenter_AssistedFactory(Provider<Long> foo) {
+    this.foo = foo;
+  }
+  @Override public MyPresenter create(String bar) {
+    return new MyPresenter(foo.get(), bar);
+  }
+}
+```
+
+Usage with Dagger2
+============
+In order to allow Dagger to use the generated factory, define an assisted dagger module:
+```java
+  @AssistedModule
+  @Module(includes = AssistedInject_PresenterModule.class)
+  abstract class PresenterModule {}
+```
+The library will generate the `AssistedInject_PresenterModule` for us. 
+
+Download
+============
+```groovy
+compileOnly 'com.squareup.inject:assisted-inject-annotations:0.1.1'
+kapt 'com.squareup.inject:assisted-inject-processor:0.1.1'
```

Comment:
I am not much familiar with the usage requirements around `kapt` and `annotationProcessor` but I couldn't get this working without `kapt`.
Using `annotationProcessor` didn't seem to generate them and Dagger complained about creating providers for the factories.

Reply:
`kapt` is only required if you are using Kotlin and Kotlin users will be aware of that. Since the reset of the README is all about Java, I don't think this one section should suddenly deviate.